pom.xml

```diff
@@ -99,21 +99,21 @@
 
     <properties>
         <!--  Java EE specs  -->
-        <cdi-version>2.0</cdi-version>
-        <jaxrs-version>2.1</jaxrs-version>
-        <jsonb-version>1.0</jsonb-version>
-        <jsonp-version>1.1</jsonp-version>
-        <annotation-version>1.3</annotation-version>
+        <cdi-version>[2.0,2.1)</cdi-version>
```

Comment:
I don't think we discussed version ranges for base specifications coming from Java EE/Jakarta EE. I'd be wary of adding it for the base specs

Reply:
Yes, I wondered about this too. I had a rummage then wrote:
"Both Jakarta and Microprofile use this convention."
(because they do ) but I agree it is beyond what we discussed - if the concensus is that, for example,  2.0.*
benefits are not worth the risk for Jakarta then I can remove it but 
I think the other way. I don't think Jakarta spec leads are going to make breaking
spec changes as 3rd digit changes and I don't think we should treat Jakarta as less
trust worthy in any governance issue like that. I don't want to anyway.

Reply:
I have removed version changes from the base javax specs.